### ISSUE TITLE
Reimplement RequestLog on the server

### ISSUE THREAD

**Issue by qdot** [buttplugio/buttplug-rs#131]
**Describe the bug**

When implementing the tracing logging in the server, the RequestLog method was never reimplemented. It just returns true without actually setting up the logging layer.

**Comment by qdot** [comment]
... Not actually sure I want to do this? Having logging in the protocol is handy for remote log correlation, but it's also a huge information leak from the server to a client we may or may not trust. For cases where the client/server are in the same process, or even same machine, this just seems silly.

Right now, on any RequestLog message, I'm going to return a log message that just says "logging not implemented" while I consider this. Between the logging we have in the client and server separately, we should be able to get along ok for now?

Basically, kicking this can down the road a bit until I figure out whether this is a thing that should even be in the protocol.

cc @blackspherefollower

**Comment by blackspherefollower** [comment]
I agree: if Intiface can control the server's log level, it probably doesn't need to come over the wire. Assuming of course there's a server.

For embedded use, there's probably still a need to set the log level and recieve logs, but not necessarily across the Buttplug protocol. Similarly, client log handling will probably still be needed too.

I suspect you're already doing something to address the real use cases across FFI, so in the context of the actual Buttplug protocol, deprecation seems reasonable.

**Comment by qdot** [comment]
Yup, I got log exporting into buttplug-js and wired through intiface-cli-rs to Desktop yesterday. I want to land that chain then add a log export callback to the FFI layer, as well as a log level output setting method. It'll just spew across text JSON blocks for now while I'm figuring things out (Yes, I realize I'm ferrying JSON across flatbuffers, may change this to a flatbuffer struct itself in the future, once I have a better idea of what all this looks like in action).

The fun part about this is that the new tracing based logging in buttplug-rs, combined with per-task instrumentation, will allow us to be able to look at the session logs for the connection of a single toy, versus the chronologically sorted pile of messages we get right now.

**Comment by qdot** [comment]
RequestLog/Log are officially deprecated, so I'm removing these APIs in v0.10. Message impls will remain for older systems, but requesting logs will just return an "Unhandled message" reply.
